test/common/json/config_schemas_test.cc

```diff
@@ -0,0 +1,69 @@
+#include "common/json/config_schemas.h"
+#include "common/json/json_loader.h"
+
+#include "test/test_common/environment.h"
+
+#include <dirent.h>
+
+using testing::_;
+
+namespace Json {
+
+std::vector<std::string> generateTestInputs() {
```

Comment:
This is basically the same code that is used in example_config_tests. It might be worth it to factor it out so that we can use it in both places (basically func to return a list of JSON that can be fed into TEST_P).